tdmpc2/common/layers.py

```diff
@@ -32,13 +32,13 @@ class ShiftAug(nn.Module):
 	def __init__(self, pad=3):
 		super().__init__()
 		self.pad = pad
+		self.padding = tuple([self.pad] * 4)
```

Comment:
quality: I don't think we should re-compute this every time. Negligeable overhead but still...